fedmsg/meta/base.py

```diff
@@ -168,6 +168,13 @@ def long_form(self, msg, **config):
         """ Return some paragraphs of text about a message. """
         return ""
 
+    def lexer(self, msg, **config):
+        """ Return a pygments lexer that can be applied to the long_form.
+
+        Returns None if no lexer is associated.
+        """
+        return None
```

Comment:
I had that originally, but I couldn't find a sensible default.  Using None works as long as the caller checks for it.